hoomd/hpmc/OBB.h

```diff
@@ -115,7 +115,7 @@ struct OBB
         is_sphere = 1;
         }
 
-    DEVICE OBB(const hoomd::detail::AABB& aabb)
+    DEVICE explicit OBB(const hoomd::detail::AABB& aabb)
```

Comment:
This change ensures that no future automatic conversions from AABB to OBB are allowed.

Callers must opt into the conversion explicitly: `OBB obb(aabb)` or `OBB(aabb)`.